api/v1/routes/forms/{id}/submission/new.js

```diff
@@ -0,0 +1,41 @@
+export default function () {
+  let operations = {
+    GET
+  };
+
+  function GET(req, res, next) {
+    const submission = {
+      id: req.params.id,
+      submission: "My form response",
+    }
+    res.status(200).json(submission);
+  }
+
+  GET.apiDoc = {
+    summary: "Retrieves the oldest submission with status ‘new’ and modifies status to ‘downloaded’",
```

Comment:
Just left a comment in Design Doc 19.  With the issues we're seeing around the indexes in DynamoDB we may have to change the behavior of this endpoint.